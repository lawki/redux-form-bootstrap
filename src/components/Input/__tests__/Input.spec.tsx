import * as React from "react";
import { shallow } from "enzyme";
import { Input } from "..";
describe("Input", () => {
  it('should render correctly', () => {
    const component = shallow(<Input />);
    expect(component).toMatchSnapshot();
  });
});
