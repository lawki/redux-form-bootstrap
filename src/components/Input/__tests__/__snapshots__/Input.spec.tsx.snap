// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render correctly 1`] = `
<Fragment>
  <InputGroup
    tag="div"
  >
    <Input
      onChange={[MockFunction]}
      type="text"
    />
  </InputGroup>
</Fragment>
`;
